internal/fs/wrappers/monitoring.go

```diff
@@ -46,6 +46,26 @@ var (
 	tracer = otel.Tracer(name)
 )
 
+// Error categories
+const (
+	deviceErr         = "DEVICE_ERRORS"
```

Comment:
Could we directly use "DEVICE_ERRORS" or "FILE_EXISTS" instead of deviceErr, dirNotEmptyErr

Reply:
It goes against the styleguide: https://google.github.io/styleguide/go/decisions#constant-names

Reply:
Ohh Thanks Kislay for this. 
I was thinking how to make this more readable. Could we prefix this with err instead brings a bit of consistency to make it more readable.
Could we then keep it errDevice, errFileNotFound, errUnimplemented, etc.

Reply:
this is more of a nit at this point. Either way upto you if you want to take it or keep it same.